src/main/java/com/sda/diary/Diary.java

```diff
@@ -8,10 +8,31 @@
 
 public class Diary {
 
-    private List<Entry> entries = new ArrayList<>();
+    public List<Entry> entries = new ArrayList<>( );
```

Comment:
Nie jestem pewny gdzie uczynienie tej zmiennej publiczną jest potrzebne, ale trochę to łamie enkapsulację - nie chcielibyśmy by użytkownicy mogli nam mieszać we wpisach. :)